syntax = "proto3";

message Message {
    string message = 1;
}
service FLOrchestrator {
    rpc PollExperiment(PollExperimentRequest) returns (PollExperimentResponse) {}
    rpc OpenStream(stream Message) returns (stream Message) {}
}



message PollExperimentRequest {
    string node = 1;
}

message PollExperimentResponse {
    string experiment_name = 1;
}